Update sample.md (#7244)
Particular/docs.particular.net: samples/bridge/sql-multi-instance/sample.md

snippet: ReceiverConfiguration

> Note that the endpoint configuration contains no routing information. The response message is a regular reply. NServiceBus, together with the bridge, handles the routing of reply messages. The receiver replies with `ClientOrderResponse` back to the sender.

Reviewer: ```suggestion
> [!NOTE]
> The endpoint configuration contains no routing information. The receiver respond to the sender with a `ClientOrderResponse` message using the [Reply method](https://docs.particular.net/nservicebus/messaging/reply-to-a-message). Using this method, NServiceBus, together with the bridge, handles the routing of reply messages without any additional configuration.
```
